docs/content/4.api/2.composables/6.use-content-head.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'useContentHead()'
+description: 'Configuring your <head> tag from your content has never been easier!'
+---
+
+`useContentHead()`{lang="ts"} is a small helper providing easy binding between your content data and [`useHead`](https://v3.nuxtjs.org/guide/features/head-management) composable from Nuxt 3.
+
+It is already implement for you in both [`<ContentDoc />`](/api/components/content-doc) component and the default [`documentDriven`](https://content.nuxtjs.org/guide/writing/document-driven) catch-all page.
```

Comment:
Would be nice to add a sentence to explain the binding done and move this section from https://content.nuxtjs.org/guide/writing/markdown here

![Screenshot 2022-06-27 at 16 54 03](https://user-images.githubusercontent.com/904724/175970627-e8e2e0b4-4277-4c1c-8e2e-5447ffff5482.png)

Also linking from the Markdown and Documentdriven to this page for head management defaults.

Reply:
I improved the documentation on both sections a lot in `8f12c6a`